### PR TITLE
Two small patches

### PR DESCRIPTION
**Pull request by cgwalters**
Going to start batching some small patches to hopefully get through e2e-aws as a group.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,59 @@ View operator status:
...
+        mode: 420
+        path: /etc/chrony.conf
+```
+
+The controller will notice the new MachineConfig and generate a new
+"rendered" version that looks like `worker-<hash>`.  Use
+`oc describe machineconfigpool/worker` to monitor the status of the rollout
+of the new rendered config to each node.
+
+Note this configuration only applies to workers; currently if
```
Comment (jlebon) [inline README.md]: Maybe put e.g. "(see the `role` label)" to be explicit about why it only applies to workers.

File: README.md [hunk 2]
```diff
@@ -38,7 +38,59 @@ View operator status:
...
+obviously a quite large number of things one may want to configure on
+a system.  For example, offline environments may want to specify
+an internal NTP pool.  Another example is static network configuration.
+By providing a MachineConfig object containing Ignition, arbitrary
+files can be laid down into `/etc` and `/var`.
+
+One known ergonomic issue right now is that you must encode file contents
+via [`data:` URIs](https://en.wikipedia.org/wiki/Data_URI_scheme). This is part
+of the specification of the [Ignition project](https://github.com/coreos/ignition).
+
```
Comment (jlebon) [inline README.md]: Might be worth also mentioning we support laying down systemd units, rather than falling back to writing the units as files.

**Comment by jlebon** [comment]
Just some optional bits, looks good to me otherwise!
/approve

**Comment by cgwalters** [comment]
OK tweaked

**Comment by ashcrow** [comment]
Monitor flake.

/retest

**Comment by ashcrow** [comment]
TF flake

/retest

**Pull request merged**